src/checkers/inference/solver/backend/lingeling/LingelingSolver.java

```diff
@@ -33,7 +33,10 @@
 public class LingelingSolver extends MaxSatSolver {
 
     // Lingeling binary executable file should be located at JSR308/lingeling/lingeling.
-    private final String lingeling = System.getenv().get("JSR308") + "/lingeling/lingeling";
+    private String currentDirPath = System.getProperty("user.dir");
+    private String JSR308 = currentDirPath.substring(0, currentDirPath.lastIndexOf("checker-framework-inference/"));
```

Comment:
is the final slash here required?

Reply:
I think it can be removed, it's useless.